### PR TITLE
Api docs from release build

### PR DESCRIPTION
**Pull request by bettinaheim**

### CODE REVIEW

File: api/qsharp/microsoft.quantum.canon.applyquantumfouriertransformbe.yml [hunk 1]
```diff
@@ -3,11 +3,7 @@ uid: microsoft.quantum.canon.applyquantumfouriertransformbe
 name: ApplyQuantumFourierTransformBE
 type: operation
 namespace: Microsoft.Quantum.Canon
-summary: |-
-  > [!WARNING]
-  > Deprecated
-
-  This operation has been removed.
+summary: ApplyQuantumFourierTransformBE has been deprecated. Please use @"applyquantumfouriertransform(bigendianaslittleendian(qs))" instead.
```
Comment (cgranade) [inline api/qsharp/microsoft.quantum.canon.applyquantumfouriertransformbe.yml]: It looks like something went wrong with the deprecation summary here, in that `applyquantumfouriertransform(bigendianaslittleendian(qs))` isn't a valid UID.

File: api/qsharp/microsoft.quantum.canon.applyripplecarrycomparatorbe.yml [hunk 2]
```diff
@@ -3,11 +3,7 @@ uid: microsoft.quantum.canon.applyripplecarrycomparatorbe
 name: ApplyRippleCarryComparatorBE
 type: operation
 namespace: Microsoft.Quantum.Canon
-summary: |-
-  > [!WARNING]
-  > Deprecated
-
-  This operation has been removed.
+summary: ApplyRippleCarryComparatorBE has been deprecated. Please use @"compareusingripplecarry(bigendianaslittleendian(x), bigendianaslittleendian(y), output)" instead.
```
Comment (cgranade) [inline api/qsharp/microsoft.quantum.canon.applyripplecarrycomparatorbe.yml]: Similarly here.

File: api/qsharp/microsoft.quantum.canon.assertprobintbe.yml [hunk 3]
```diff
@@ -3,11 +3,7 @@ uid: microsoft.quantum.canon.assertprobintbe
 name: AssertProbIntBE
 type: operation
 namespace: Microsoft.Quantum.Canon
-summary: |-
-  > [!WARNING]
-  > Deprecated
-
-  This operation has been removed.
+summary: AssertProbIntBE has been deprecated. Please use @"assertprobint(stateindex, prob, bigendianaslittleendian(qubits), tolerance)" instead.
```
Comment (cgranade) [inline api/qsharp/microsoft.quantum.canon.assertprobintbe.yml]: Ditto.

File: api/qsharp/microsoft.quantum.canon.applyquantumfouriertransformbe.yml (hunk 1 above)
Comment (bettinaheim) [reply api/qsharp/microsoft.quantum.canon.applyquantumfouriertransformbe.yml]: Ok, there should still be a valid link, so I suggest to take it in here. I'll file a task under the general clean-up deliverable.

**Pull request merged**